app/uk/gov/hmrc/tariffclassificationfrontend/connector/BindingTariffClassificationConnector.scala

```diff
@@ -59,9 +59,9 @@ class BindingTariffClassificationConnector @Inject()(appConfig: AppConfig, clien
     s"${appConfig.bindingTariffClassificationUrl}/cases?$queryString"
   }
 
-  def findCasesByQueue(queue: Queue, pagination: Pagination)(implicit hc: HeaderCarrier): Future[Paged[Case]] = {
+  def findCasesByQueue(queue: Queue, pagination: Pagination, types: Seq[ApplicationType] = Seq(ApplicationType.BTI, ApplicationType.LIABILITY_ORDER))(implicit hc: HeaderCarrier): Future[Paged[Case]] = {
     val queueId = if (queue == Queues.gateway) "none" else queue.id
-    val url = buildQueryUrl(withStatuses = statuses, queueId = queueId,  assigneeId = "none", pagination = pagination)
+    val url = buildQueryUrl(types = types, withStatuses = statuses, queueId = queueId,  assigneeId = "none", pagination = pagination)
```

Comment:
should `withStatuses` just be `statuses`?
Or prefix them all with `with`?

Reply:
Should be just statuses really as withStatuses suggests a builder type pattern which it isn't. I think I will try to turn this into a Search object anyway as mentioned previously